### PR TITLE
feat(be:FSADT1-1344): database changes to support new version of the FCMS - Postgres

### PR DESCRIPTION
**Pull request by mamartinezmejia**
---

Thanks for the PR!

Deployments, as required, will be available below:
Any successful deployments (not always required) will be available [here](https://nr-forest-client-47-frontend.apps.silver.devops.gov.bc.ca/)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-forest-client/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-forest-client/actions/workflows/merge.yml)

### CODE REVIEW

File: backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
+alter table nrfc.submission_detail
+add column if not exists work_safe_bc_number varchar(6) null,
+add column if not exists doing_business_as varchar(120) null,
+add column if not exists client_acronym varchar(8) null,
+add column if not exists TBD_ID_TYPE varchar(8) null,
+;
+
+comment on column nrfc.submission_detail.work_safe_bc_number is 'TBD';
```
Comment (paulushcgcj) [inline backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql:8]: Shouldn't we add a proper comment here? If there's no info to be added, is better to remove it at all

File: backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql
Comment (paulushcgcj) [inline backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql:1]: Make sure the file name format is V#__name.sql. It looks like your file name has a single _ after the version

File: backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+comment on column nrfc.submission_detail.doing_business_as is 'TBD';
+comment on column nrfc.submission_detail.client_acronym is 'TBD';
+
+alter table nrfc.submission_contact
+add column if not exists middle_name varchar(30) null,
```
Comment (paulushcgcj) [inline backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql:13]: The middle name was dropped from contact. Please remove it

File: backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql [hunk 3]
```diff
@@ -0,0 +1,21 @@
+alter table nrfc.submission_detail
+add column if not exists work_safe_bc_number varchar(6) null,
+add column if not exists doing_business_as varchar(120) null,
+add column if not exists client_acronym varchar(8) null,
+add column if not exists TBD_ID_TYPE varchar(8) null,
```
Comment (paulushcgcj) [inline backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql:5]: If this is not defined yet, don't add it

**Review by paulushcgcj — CHANGES_REQUESTED** [review]

File: backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql (hunk 1 above)
Comment (mamartinezmejia) [reply backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql:8]: Yes, this is a draft. I am working on it. I will request your review once I am done.

File: backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql
Comment (mamartinezmejia) [reply backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql:1]: Yes. Thanks.

File: backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql (hunk 2 above)
Comment (mamartinezmejia) [reply backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql]: Yes, I made that before leaving. I will change it. I will request your review once I am done with the draft.

File: backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql (hunk 3 above)
Comment (mamartinezmejia) [reply backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql]: This is a draft. I will put the right names shortly.

**Review by paulushcgcj — APPROVED** [review]

**Pull request merged**